src/test/fuzz/utxo_snapshot.cpp

```diff
@@ -160,7 +160,7 @@ void utxo_snapshot_fuzz(FuzzBufferType buffer)
             const auto* index{chainman.m_blockman.LookupBlockIndex(block->GetHash())};
             Assert(index);
             Assert(index->nTx == 0);
-            if (index->nHeight == chainman.GetSnapshotBaseHeight()) {
+            if (index->nHeight == chainman.CurrentChainstate().SnapshotBase()->nHeight) {
```

Comment:
In commit 529890d836cb75175c7943ee0a79c536412d002c

I get a compilation error here. The change seems to be two commits too early, but the line also needs to be changed in this commit, so there would be some line churn.